development-vm/Procfile

```diff
@@ -142,7 +142,6 @@ draft-router-api:            govuk_setenv draft-router-api            ./run_in.s
 # firewall block rule on port 3170 (for testing connect timeouts)
 # canary-frontend has reserved port 3200
 # canary-backend has reserved port 3201
-# kibana has reserved port 3202
```

Comment:
I think we leave old used ports to avoid them being reused in the future just in case. For example, above I can see we've still got `multipage-frontend used port 3118` and `share-sale-publisher used port 3119`.

Reply:
Updated